mycore-base/src/main/java/org/mycore/access/MCRAccessBaseImpl.java

```diff
@@ -35,13 +35,13 @@
 
 /**
  * This class is a base implementation of the <code>MCRAccessInterface</code>.
- * 
+ *
  * It will simply allow everything and will do nothing on persistent operations.
  * Feel free to extend this class if your implementation can only support parts
  * of the Interface definition.
- * 
+ *
  * @author Jens Kupferschmidt
- * 
+ *
```

Comment:
You have a blank line at the end of each javadoc comment. Looks strange.

Also in between the javadoc you added some. Intellij doesn't like it: "Blank line will be ignored -> Insert &lt;p&gt;."

Reply:
Comes from IntelliJ. It automatically removes blanks at end of line on save. Do not know how to deactivate it, but those "space" are not needed. That's why I didn't put much effort in it.